README.rst

```diff
@@ -1,10 +1,10 @@
 West Oakland Environmental Indicators Project website  |Travis|_ |Codecov|_
-===================================================
-.. |Travis| image:: https://travis-ci.org/TODO/woeip.svg?branch=master
-.. _Travis: https://travis-ci.org/TODO/woeip
+===========================================================================
+.. |Travis| image:: https://travis-ci.org/openoakland/woeip.svg?branch=master
```

Comment:
This change corrects the URLs for the Travis (testing) and Codecov (test coverage reporting) badges.